check_test.go

```diff
@@ -346,31 +349,31 @@ func getCheckTestCases() map[string]TmpCheckTestCase {
 			fixture: BuildUpdateRequest("CodeIssue", map[string]any{
 				"constraint":     ol.CheckCodeIssueConstraintEnumContains,
 				"issueName":      "code-issue-updated",
-				"issueType":      nil, // API NOTE - not allowed when constraint is "contains"
+				"issueType":      &[]string{}, // API NOTE - not allowed when constraint is "contains"
```

Comment:
shouldn't this be `(*[]string)(nil)` to be the same as before?  I guess i would expect `&[]string{}` to be an initialized by empty slice and thus generating `[]` ?